frontend/packages/dev-console/src/components/helm/HelmInstallUpgradePage.tsx

```diff
@@ -115,9 +119,22 @@ const HelmInstallUpgradePage: React.FunctionComponent<HelmInstallUpgradePageProp
 
     config
       .fetch('/api/helm/release', payload)
-      .then(() => {
-        actions.setSubmitting(false);
-        history.push(config.redirectURL);
+      .then(async (res: HelmRelease) => {
+        let redirect = config.redirectURL;
+
+        if (helmAction === HelmActionType.Install && res?.info?.notes) {
+          const labelSelector = `name=${res.name},version=${res.version},owner=helm`;
+          const secret = await k8sGet(SecretModel, null, res.namespace, {
+            queryParams: { labelSelector },
+          });
+          const secretId = _.get(secret, 'items.0.metadata.uid', '');
+          if (secretId) {
+            redirect = `${config.redirectURL}?selectId=${secretId}&selectTab=Release+Notes`;
+          }
```

Comment:
I'd wrap this in a try catch because if it fails, we may as well continue to redirect to because the install was still successful.

Reply:
Done.